src/TodoPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import TodoList from './TodoList';
+import TodoAddBox from './TodoAddBox';
+import EmptyBox from './EmptyBox';
+
+function TodoPage({
+  todo, todos, isEmpty, handleChangeTodo, handleClickAdd, handleClickFinish,
+}) {
+  return (
+    <div>
+      <h2>To-do</h2>
+      <TodoAddBox
+        todo={todo}
+        handleChangeTodo={handleChangeTodo}
+        handleClickAdd={handleClickAdd}
+      />
+      <EmptyBox isEmpty={isEmpty} />
+
+      <TodoList
+        todos={todos}
+        handleClickFinish={handleClickFinish}
+      />
```

Comment:
```suggestion
      {isEmpty ? <EmptyBox /> : (
        <TodoList
          todos={todos}
          handleClickFinish={handleClickFinish}
        />
      )
```
리스트가 비어있을때 EmptyBox가 노출되고 아닐 경우, List가 노출되도록 이렇게 처리하는건 어떤가요?

Reply:
조건에 따라 컴포넌트가 실행되는 방식이  더 좋은거 같습니다.! 

1.  EmptyBox와 TodoList 두개가 모두 보일 확률이 없어집니다.
2. 굳이 실행되지 않아도 되는 컴포넌트가 실행된 후 그 안에서 조건을 확인하니 비효율 적인거 같습니다.